markdown/api-docs/catalog/products-overview.md

```diff
@@ -540,6 +541,21 @@ lineNumbers: true
 * Must have a SKU code (unless they’re a base variant)
 * In the case of non-base variants, variants will relate to a particular combination of variant option values - such as “small” and “red”
 
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--warning">
+<div class="HubBlock-content">
+    
+<!-- theme: warning -->
+
+### V2 SKU rules will override Variant pricing
+Creating SKU rules via the V2 API or via CSV import will override any Variant sale pricing added to a product via the Control Panel, V3 API or Price Lists UI.

```

Comment:
Body missing `>` 

```
> Creating SKU rules via the V2 API or via CSV import will override any Variant sale pricing added to a product via the Control Panel, V3 API or Price Lists UI.
```

Also add this to https://developer.bigcommerce.com/legacy/v2-products/v2-v3 Products in V3
s > Variants.

Reply:
Added